packages/icon-builder/package.json

```diff
@@ -1,12 +1,17 @@
 {
-  "name": "material-ui-icon-builder",
-  "version": "0.1.2",
-  "description": "Material Design Svg Icons converted to React components.",
   "private": "true",
-  "scripts": {
-    "prebuild": "rimraf js jsx",
-    "build": "node build.js --output-dir ../../src/svg-icons --svg-dir ./node_modules/material-design-icons --glob '/**/production/*_24px.svg' --mui-require relative --renameFilter ./filters/rename/material-design-icons.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+  "name": "material-ui-icon-builder",
```

Comment:
You are right, but the branding, documentation make me think otherwise:
> This **tool** generates Material-UI SvgIcon components for a set of svg icons.